client/lib/signup/flow-controller.js

```diff
@@ -159,7 +159,21 @@ assign( SignupFlowController.prototype, {
 	},
 
 	_processStep: function( step ) {
-		const dependencies = steps[ step.stepName ].dependencies || [];
+		let dependencies = steps[ step.stepName ].dependencies || [];
```

Comment:
Super minor, but this should still be `const` because you aren't reassigning to `dependencies`.